src/components/PasswordForm/style.less

```diff
@@ -1,49 +1,50 @@
 @import '../../styles/fonts.less';
 @import '../../styles/vars.less';
 
-.passwordcard {
-  align-items: center;
-  background: @blue-2;
-  border-radius: 10px;
-  color: @white;
-  display: flex;
-  flex-direction: column;
-  font-family: 'Nunito', sans-serif;
-  height: @password-height;
-  justify-content: center;
-  width: @sign-in-width;
+.password-card {
+  padding-top: 64px;
 
-  h1 {
-    color: @white;
-    font-size: @card-heading-2;
-    margin-top: 24px;
+  .title {
+    color: @black;
+    font-size: 48px;
+    font-weight: 600;
+    margin: 24px 0;
+    text-align: center;
   }
 
-  label {
-    color: @white;
-    font-family: 'Nunito';
-    font-size: @general-body;
+  .form {
+    display: flex;
+    flex-direction: column;
   }
 
-  .logo {
-    height: 115px;
-    margin-bottom: 10px;
-    width: 115px;
-  }
+  .email-container {
+    position: relative;
+
+    .icon {
+      left: 8px;
+      position: absolute;
+      top: 6.5px;
+    }
 
-  .input-box {
-    height: @button-height-2;
-    margin-right: 10px;
-    width: 200px;
+    .input-box {
+      border: none;
+      border-bottom: 1px solid @black;
```

Comment:
The login input element's font size seems a bit too small, it looks better with `1rem` instead of `1em`

![image](https://user-images.githubusercontent.com/32887362/95108481-2bb2fc00-06f0-11eb-9ab9-db0278d95779.png)